### PR TITLE
Add missing dependencies

### PR DESCRIPTION
**Pull request by jderusse**

### CODE REVIEW

File: src/Integration/Symfony/Bundle/composer.json [hunk 1]
```diff
@@ -37,6 +37,5 @@
         "psr-4": {
             "AsyncAws\\Symfony\\Bundle\\Tests\\": "tests/"
         }
-    },
-    "minimum-stability": "dev"
```
Comment (Nyholm) [inline src/Integration/Symfony/Bundle/composer.json:41]: If we keep this line, we dont need to make another PR to remove `@dev` before releasing a new version.

Comment (jderusse) [reply src/Integration/Symfony/Bundle/composer.json:41]: it was for consistency with FlySystem integration. don't we have to create PR to add `^0.n` for each minor version (before 1.0)?

Comment (Nyholm) [reply src/Integration/Symfony/Bundle/composer.json:41]: This line was removed from Flysystem because it was downloading dev-master from flysystem v2. That broke CI time to time. 

> don't we have to create PR to add ^0.n for each minor version (before 1.0)?

We do. But I dont want to make a PR to remove `@dev` before a new release.

Comment (jderusse) [reply src/Integration/Symfony/Bundle/composer.json]: reverted
